source/includes/_event_triggered_survey.md

```diff
@@ -0,0 +1,34 @@
+#Event Triggered Survey
+
+```javascript
+// You define a global variable to track your events
+my_global_event_name = 'loaded_website'; 
+
+// You define your WootricSettings variable at the beginning of the code
+wootricSettings = {
+  account_token: 'NPS-YOURTOKEN',
+  email: 'user@example.com',
+  created_at: 1528416000,
+  properties: {
+    favorite_color: 'blue'
+  },
+  event_name: my_event
```

Comment:
`my_event` vs `my_global_event_name`

Reply:
I think you should delete this since you are referencing the `event_name` key bellow. We can add a comment for our users to write into that variable when they wish to track an event instead of duplicating it with `my_global_event_name`. What do you think?